code/association_scan/TensorQTL/TensorQTL.ipynb

```diff
@@ -52,17 +52,17 @@
     "\n",
     "    The header of the bed.gz is per the [TensorQTL](https://github.com/broadinstitute/tensorqtl) convention:\n",
     "\n",
-    "    >    Phenotypes must be provided in BED format, with a single header line starting with # and the first four columns corresponding to: chr, start, end, phenotype_id, with the remaining columns corresponding to samples (the identifiers must match those in the genotype input). The BED file should specify the center of the cis-window (usually the TSS), with start == end-1.\n",
+    "    >    Phenotypes must be provided in BED format, sorted by chromosome and [start,end] position, with a single header line starting with # and the first four columns corresponding to: chr, start, end, phenotype_id, with the remaining columns corresponding to samples (the identifiers must match those in the genotype input). The BED file should specify the cis-window (usually the TSS), with start = the minimum start for each gene, end = the maximum end for each gene(extracted from phenotype referenced gtf file).\n",
     "\n",
     "\n",
-    "- List of genotypes in PLINK binary format (`bed`/`bim`/`fam`) for each chromosome, previously processed through our genotype QC pipelines.\n",
+    "- List of genotypes in both PLINK binary format (`bed`/`bim`/`fam`) and PLINK 2 binary genotype table (`pgen`/`pvar`/`psam`) for each chromosome, previously processed through our genotype QC pipelines.\n",
     "- Covariate file, a file with #id + samples name as colnames and each row a covariate: fixed and known covariates as well as hidden covariates recovered from factor analysis.\n",
     "- Optionally, a list of traits (genes, regions of molecular features etc) to analyze.\n",
     "\n",
     "\n",
     "For cis-analysis:\n",
     "\n",
-    "- Optionally, a list of genomic regions associate with each molecular features to analyze. The default cis-analysis will use a window around TSS. This can be customized to take given start and end genomic coordinates."
+    "- Optionally, a list of genomic regions associate with each molecular features to analyze. The default cis-analysis will use a window around TSS. This can be customized to take given start and end genomic coordinates. We have currently disabled the use of a customized cis window for cis_association, as it did not yield significant improvements. "
```

Comment:
> We have currently disabled the use of a customized cis window for cis_association

Should say that we currently suggest using 1Mb window around a gene because longer customized cis-windows (such as extending by TAD) does not yield significant improvements.